prac_10/wiki.py

```diff
@@ -0,0 +1,13 @@
+import wikipedia
+
+user_search = input("Enter your search word here: ")
+if user_search == "":
+    print("Search has ended. Try again!")
+else:
+    try:
+        user_search = wikipedia.page(user_search)
+    except wikipedia.exceptions.DisambiguationError as e:
+        print(e.options)
+    print(user_search.title)
+    print(user_search.summary)
+    print(user_search.url)
```

Comment:
Add some descriptive text in your print statement for the wiki search eg "Page Title: {user_search.title}"